scripts/stubsabot.py

```diff
@@ -200,10 +200,11 @@ def all_py_files_in_source_are_in_py_typed_dirs(source: zipfile.ZipFile | tarfil
     if not all_python_files:
         return False
 
-    for path in all_python_files:
-        if not any(py_typed_dir in path.parents for py_typed_dir in py_typed_dirs):
-            return False
-    return True
+    return all(
+        # fmt: skip
```

Comment:
Why is this needed? This looks like how Black would format the code anyway.

Reply:
With our 130 line length black puts the whole thing on a single line. Which greatly affected readability. (Black and Ruff don't tend to have great vertical formatting for ternaries and comprehensions).

For the sake of discussion about the "reimplemented-builtin (SIM110)" rule, I worked around this formatting issue

Reply:
If you need to manually format the result, that seems like an argument against applying this "simplification" rule.